comps/dataprep/qdrant/prepare_doc_qdrant.py

```diff
@@ -47,11 +48,18 @@ def ingest_data_to_qdrant(doc_path: DocPath):
 
     content = document_loader(path)
 
-    chunks = text_splitter.split_text(content)
+    structured_types = [".xlsx", ".csv", ".json", "jsonl"]
```

Comment:
the structured_types are used by many files, we should define it as common variable.